Index: kubernetes/build/lib/release.sh
===================================================================
--- kubernetes.orig/build/lib/release.sh
+++ kubernetes/build/lib/release.sh
@@ -294,7 +294,7 @@ function kube::release::create_docker_im
     local docker_tag="${KUBE_GIT_VERSION/+/_}"
     if [[ -z "${docker_tag}" ]]; then
       kube::log::error "git version information missing; cannot create Docker tag"
-      return 1
+      docker_tag='v1.7.10'
     fi
 
     for wrappable in "${binaries[@]}"; do
@@ -349,7 +349,7 @@ function kube::release::create_docker_im
           kube::log::status "Deleting docker image ${docker_image_tag}"
           "${DOCKER[@]}" rmi ${docker_image_tag} &>/dev/null || true
         fi
-      ) &
+      )
     done
 
     if [[ "${KUBE_BUILD_HYPERKUBE}" =~ [yY] ]]; then
